package.json

```diff
@@ -64,14 +61,15 @@
     },
     "browserslist": {
         "production": [
-            ">0.2%",
+            ">0.5%",
```

Comment:
What does this browserlist thing do and why are we changing it?

Reply:
It's for compatibility. 
https://mui.com/material-ui/migration/migration-v4/#supported-browsers-and-node-versions

It tells React, which browsers we are targeting, A newer target would mean less polyfills or legacy code and faster and smaller bundles.